Sources/swift-format/Frontend/Frontend.swift

```diff
@@ -176,35 +177,80 @@ class Frontend {
     at configurationFileURL: URL?,
     orInferredFromSwiftFileAt swiftFileURL: URL?
   ) -> Configuration? {
-    // If an explicit configuration file path was given, try to load it and fail if it cannot be
-    // loaded. (Do not try to fall back to a path inferred from the source file path.)
+    // The actual config file that this frontend is going to use.
+    let configURL: URL?
```

Comment:
I need the config file URL later to figure out the source location, so this function got a little refactor in.